lib/apiHelpers.ts

```
@@ -0,0 +1,154 @@
+
+    if (env === "prod") {
+      try {
+        // await indexEvent(event);
+      } catch (err) {
+        console.error(`Error occurred while indexing event: ${err instanceof Error ? err.message : String(err)}`);
+        captureException(err, {
+          extra: {
+            context: 'indexing event after Google Calendar update'
+          }
+        });
+      }
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)